_includes/google-analytics.html

```diff
@@ -4,8 +4,10 @@
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-  ga('create', '{{ site.google_analytics }}', 'auto');
+  ga('create', '{{ site.google_analytics.tracking_id }}', 'auto');
+  {% if site.google_analytics.anonymize_ip == true %}
+    ga('set', 'anonymizeIp', true);
+  {% endif %}
```

Comment:
@philipwalton thanks for the detailed answer 😄 

@parkr So using your code snippet would imply asking users to check [analytics field references](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference) to see what properties they can set. A bit harsh for the majority of users don"t you think?